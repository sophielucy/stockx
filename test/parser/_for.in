for (x = 3; x < 10; ) {
    print("why do meteors always land in craters?");
}
