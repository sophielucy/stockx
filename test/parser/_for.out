For(Assign(x, IntLiteral(3)); Binop(Id(x), Less, IntLiteral(10)); Noexpr) { Block(Call(print, StringLiteral(why do meteors always land in craters?))) }
